### ISSUE TITLE
How to download training logs to view in tensorflow?

### ISSUE THREAD

**Issue by the-it-weirdo** [ultralytics/hub#725]
### Search before asking

- [X] I have searched the HUB [issues](https://github.com/ultralytics/hub/issues) and [discussions](https://github.com/ultralytics/hub/discussions) and found no similar questions.

### Question

Hello, I trained a model with my own dataset. I used cloud training. I can see the graphs in the Ultralytics Hub. However, I am unable to add any axis title to the graph. My searches tell me that I can add axis title using tensorboard. How do I access the tranining logs for the same? How can I download the logs to open with tensorboard?

**Comment by sergiuwaxmann** [comment]
@the-it-weirdo Hello!

Unfortunately, I am not very familiar with TensorBoard but if it helps, you might be able to [get the data you need](https://pytorch.org/tutorials/recipes/recipes/tensorboard_with_pytorch.html) from the [PyTorch export](https://docs.ultralytics.com/hub/models#deploy-model).

Alternatively, you can get the metrics using [Ultralytics HUB SDK](https://docs.ultralytics.com/hub/sdk):
```python
from hub_sdk import HUBClient

# Initialize the HUBClient
credentials = {"api_key": "<API-KEY>"}
client = HUBClient(credentials)
print("Authenticated:", client.authenticated)

# Get model by ID
model = client.model("<MODEL-ID>")

# Get metrics
metrics = model.get_metrics()
print("Metrics:", metrics)

# Get weights
weight_url = model.get_weights_url("best")
print("Weight URL:", weight_url)
```
Make sure to replace `<API-KEY>` and `<MODEL-ID>` with your actual API key and model ID. This script will authenticate you, fetch the training metrics, and provide a direct link to download the best weights.
